libkbfs/block_types.go

```diff
@@ -188,10 +188,30 @@ func (fb *FileBlock) NewEmpty() Block {
 
 // DataVersion returns data version for this block.
 func (fb *FileBlock) DataVersion() DataVer {
+	// If this is an indirect block, and none of its children are
```

Comment:
I'm confused about this -- if there is a top-level indirect block with a bunch of child blocks not marked as direct, and there are no holes, can't that also be `FirstValidDataVer`?

I guess if we had a three-state type, the logic could be "if this is an indirect block, and at least one child is marked as an indirect block, then this must be a big file".